src/scene/vcPOI.cpp

```diff
@@ -1276,19 +1276,22 @@ void vcPOI::GenerateLineFillPolygon(vcState *pProgramState)
 {
   if (m_line.numPoints >= 3)
   {
+    if (m_line.pPoints[m_line.numPoints - 1] == m_line.pPoints[m_line.numPoints - 2])
```

Comment:
Why are the last points the same? I assume this crashes in the triangle generation, it might be better to explicitly check for degenerate triangles instead?

Reply:
> * Crash is still possible when crossing lines, otherwise it should be impossible.

So, are these crashes the result of 2 different problems? Or do they stem from the same issue?

Reply:
So the last 2 points will be the same while you're creating the area measurement. you click, it places a point while also having the next point ready on the mousecursor, placing both points at the exact same spot.

There is another crash due to generating the poly model, but with this fix, it should never crash unless you cross lines.

Reply:
I was able to reproduce this crash by:
- Not moving the mouse and clicking again.
- Placing two points, and moving the mouse back to the first and clicking again.

Whatever is actually crashing needs to be fixed, don't just workaround it.

Reply:
ok i know what the issue is. Ill fix it in another PR